srt.go

```diff
@@ -127,24 +133,18 @@ func ReadFromSRT(i io.Reader) (o *Subtitles, err error) {
 }
 
 // parseTextSrt parses the input line to fill the Line
-func parseTextSrt(i string) (o Line) {
+func parseTextSrt(i string, styles StyleAttributes) (Line, StyleAttributes) {
 	// special handling needed for empty line
-	if strings.TrimSpace(i) == "" {
+	o := Line{}
```

Comment:
Why remove the `TrimSpace()`? 🤔

Reply:
The input has already been trimmed on line 48, so this is unnecessary, similar to `parseTextWebVTT`.

Reply:
I do agree but we may add some more processing on the `line` variable in the future and I want to make sure that `parseTextWebVTT` does not treat spaces only as anything but an empty line 👍